src/frontend/src/pages/PIADetailPage/index.tsx

```diff
@@ -0,0 +1,187 @@
+        </div>
+        <div className="horizontal-divider"></div>
+        <div className="container d-grid gap-3">
+          <h2>
+            <b>{messages.GeneralInfoSection.H2Text.en}</b>
+          </h2>
+          <div>
+            <div className="row ">
+              <div className="col col-md-4">
+                <b>Drafter</b>
+              </div>
+              <div className="col col-md-4">
+                <b>Initiative Lead</b>
+              </div>
+              <div className="col col-md-4">
+                <b>Ministry Privacy Officer </b>
+              </div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.drafterName}</div>
+              <div className="col col-md-4">{pia.leadName}</div>
+              <div className="col col-md-4">{pia.mpoName}</div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.drafterTitle}</div>
+              <div className="col col-md-4">{pia.leadTitle}</div>
+              <div className="col col-md-4">{pia.mpoEmail}</div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.drafterEmail}</div>
+              <div className="col col-md-4">{pia.leadEmail}</div>
+            </div>
+          </div>
+          <div>
+            <div className="row">
+              <div className="col col-md-4">
+                <b>Ministry</b>
+              </div>
+              <div className="col col-md-4">
+                <b>Branch</b>
+              </div>
+            </div>
+            <div className="row">
+              <div className="col col-md-4">{pia.ministry}</div>
+              <div className="col col-md-4">{pia.branch}</div>
+            </div>
+          </div>
+        </div>
+        <div className="container pt-5 form__section">
+          <h2 className="pb-3">
+            <b>{messages.GeneralInfoSection.H2TextTwo.en}</b>
+          </h2>
+          <div>
+            <h2 className="pb-1">{messages.InitiativeDescriptionSection.en}</h2>
+
+            <div>
+              <MDEditor preview="preview" value={pia.initiativeDescription} />
+            </div>
+          </div>
+          <div>
+            <p>{messages.InitiativeScopeSection.en}</p>
+
+            <div>
+              <MDEditor preview="preview" value={pia.initiativeScope} />
+            </div>
+          </div>
+          <div>
+            <p>{messages.InitiativeDataElementsSection.en}</p>
+
+            <div>
+              <MDEditor preview="preview" value={pia.dataElementsInvolved} />
+            </div>
+          </div>
+        </div>
+        <div className="container pt-5 form__section">
+          <h2 className="pb-3">
+            <b>{messages.GeneralInfoSection.H2TextThree.en}</b>
+          </h2>
+          <div>
+            <p>{messages.InitiativePISection.en}</p>
+
+            <div>
+              <p>{pia.hasAddedPiToDataElements}</p>
```

Comment:
this should be either of the values used in the dropdown -> Yes, No, I'm not sure

and not true/false/null returned from the backend